pkg/configuration/config_manager/config_handler.go

```diff
@@ -424,6 +424,16 @@ func CreateExecHandler(command []string, mountPoint string, configMeta *ConfigSp
 	return shellTrigger, nil
 }
 
+func checkAndBackup(configMeta ConfigSpecInfo, dirs []string, filter regexFilter, backupPath string) error {
+	if isSyncReloadAction(configMeta) {
```

Comment:
why check it again inside the config manager?

Reply:
The backup configuration is to support `async reload action` in config-manager container, so it will backup a recent successful configuration version, and the latest configuration to compare, check what parameters changes;

So `sync reload action`, do not need to backup the configuration files.